dashboard/app/controllers/pd/workshop_survey_controller.rb

```diff
@@ -10,11 +10,8 @@ def new
 
     workshop = @pd_enrollment.workshop
 
-    user = @pd_enrollment.user
-    user_surveys = Pd::WorkshopSurvey.
-      joins(:pd_enrollment).
-      where(pd_enrollments: {user_id: user.id})
-    is_first_survey = user_surveys.count == 0
+    user_surveys = Pd::WorkshopSurvey.find_by_user(@pd_enrollment.user)
```

Comment:
Does this work? And if so, how? AFAICT there is no `user` field (and no `find_by_user`) in [Pd::WorkshopSurvey](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/workshop_survey.rb)